src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,22 @@
 import './MovieList.scss';
+import React from 'react';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ moviesFromServer }) => (
+  <div className="page">
+    <div className="page-content">
+      <div className="movies">
+        {moviesFromServer.map(movie => (
+          <MovieCard
+            moviesFromServer={movie}
+            key={movie.imdbId}
+          />
+        ))}
+      </div>
+    </div>
+
+    <div className="sidebar" data-cy="Sidebar">
+      Sidebar will be here
+    </div>
```

Comment:
The same. It's not connected with the MovieList component. Leave it inside the App